cmd/cluster/core/create.go

```diff
@@ -143,6 +146,8 @@ type AzurePlatformOptions struct {
 	AvailabilityZones []string
 }
 
+const managedClusterNameKey = "cluster.open-cluster-management.io/managedcluster-name"
```

Comment:
please move const declaration to the top of the file and document it, rename to managedClusterNameAnnotation and document it, e.g
```// managedClusterNameAnnotation annotates clusters....```

Also

Reply:
done